pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -198,6 +201,17 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 		}
 	}
 
+	// Check if Status is empty and status annotation is present
+	// Manually restore data.
+	if !r.hostHasStatus(host) {
+		obj := r.getHostFromAnnotation(host)
+		if obj != nil {
+			host.Status = obj.Status
+		} else {
+			reqLogger.Info("Failed to find status annotation. Couldn't restore status data manually")
```

Comment:
Since we are placing the status on the BMH when the status is missing and the annotation is present, I thought manually seems a logical word, but I will change it since it is creating confusion. And also the "Failed" message will be removed.

Reply:
Changed it